src/PlaybackManager.vala

```diff
@@ -109,8 +126,8 @@ public class Music.PlaybackManager : Object {
                     ngettext (
                         "%d track was added to the queue",
                         "%d tracks were added to the queue",
-                        files.length
-                    ).printf (files.length)
+                        files.length - invalids
```

Comment:
This is probably a super edge case, but we should check here to make sure `files.length - invalids > 0` before sending a notification since it's possible now to get a notification that says "0 tracks were added to the queue"

Reply:
OK, I'll avoid showing a notification for "0 tracks added to Q".